openshift_scalability/utils.py

```diff
@@ -350,7 +380,51 @@ def single_project(testconfig, projname, globalvars):
         pod_handler(testconfig["pods"], globalvars)
     if "rcs" in testconfig:
         rc_handler(testconfig["rcs"], globalvars)
-
+    if globalvars["autogen"]:
+        num_expected = int(globalvars["podnum"])
+        pods_running = []  
+        while len(pods_running) < num_expected: 
```

Comment:
@sjug 
If you have multiple templates in the template section, the last "podnum" will be used and this loop will stop blocking too early and not all pods will have /root/.ssh synced.  Hence the issues I hit yesterday.  So at this point, a blocker for me to use this PR's functionality unless I'm missing something.